# HG changeset patch
# User Andreas Weber <andy.weber.aw@gmail.com>
# Date 1382727583 -7200
#      Fri Oct 25 20:59:43 2013 +0200
# Node ID 92c94ba3d335c55b2ba970b8a0adc6aafc1935f2
# Parent  b14c575f79e3f8a7c4ad4164f0892e1e0ae9d985
* de_DE.ts: updated german translation

diff --git a/libgui/languages/de_DE.ts b/libgui/languages/de_DE.ts
--- a/libgui/languages/de_DE.ts
+++ b/libgui/languages/de_DE.ts
@@ -239,17 +239,17 @@ ist im Editor bereits geöffnet</translation>
     <message>
         <location line="+63"/>
         <source>&amp;Recent Editor Files</source>
         <translation>&amp;Zuletzt bearbeitete Dateien</translation>
     </message>
     <message>
         <location line="+9"/>
         <source>&amp;Edit Function</source>
-        <translation type="unfinished"></translation>
+        <translation>Funktion b&amp;earbeiten</translation>
     </message>
     <message>
         <location line="+10"/>
         <source>&amp;Close</source>
         <translation>S&amp;chließen</translation>
     </message>
     <message>
         <location line="+4"/>
@@ -259,58 +259,66 @@ ist im Editor bereits geöffnet</translation>
     <message>
         <location line="+5"/>
         <source>Close Other Files</source>
         <translation>Andere Dokumente schließen</translation>
     </message>
     <message>
         <location line="+49"/>
         <source>Run &amp;Selection</source>
-        <translation type="unfinished"></translation>
+        <translation>Markierter Quellcode au&amp;sführen</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>&amp;Help</source>
-        <translation type="unfinished">&amp;Hilfe</translation>
+        <translation>&amp;Hilfe</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>&amp;Help on Keyword</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Hilfe zu Schlüsselwort</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>&amp;Documentation on Keyword</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Dokumentation zu Schlüsselwort</translation>
     </message>
     <message>
         <location line="-147"/>
         <source>&amp;Find and Replace</source>
         <translation>&amp;Suchen und Ersetzen</translation>
     </message>
     <message>
         <location line="-584"/>
         <source>Could not open file
 %1
 for read: %2.</source>
-        <translation type="unfinished"></translation>
+        <translation>Die Datei
+%1
+konnte nicht zum Lesen geöffnet werden:
+%2.</translation>
     </message>
     <message>
         <location line="+21"/>
         <source>File
 %1
 does not exist. Do you want to create it?</source>
-        <translation type="unfinished"></translation>
+        <translation>Die Datei
+%1
+existiert nicht. Soll sie erzeugt werden?</translation>
     </message>
     <message>
         <location line="+17"/>
         <source>Could not open file
 %1
 for write: %2.</source>
-        <translation type="unfinished"></translation>
+        <translation>Die Datei
+%1
+konnte nicht zum Schreiben geöffnet werden:
+%2.</translation>
     </message>
     <message>
         <location line="+549"/>
         <source>Save File And Run</source>
         <translation>Datei speichern und ausführen</translation>
     </message>
     <message>
         <location line="+2"/>
@@ -325,22 +333,22 @@ for write: %2.</source>
     <message>
         <location line="+39"/>
         <source>&amp;Edit</source>
         <translation>&amp;Editieren</translation>
     </message>
     <message>
         <location line="+22"/>
         <source>&amp;Preferences</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Einstellungen</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>&amp;Styles Preferences</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Stil-Einstellungen</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>&amp;Debug</source>
         <translation>&amp;Debuggen</translation>
     </message>
     <message>
         <location line="+9"/>
@@ -996,17 +1004,17 @@ zu: </translation>
         <location line="-1011"/>
         <location line="+931"/>
         <source>Save Workspace As</source>
         <translation>Arbeitsumgebung speichern als</translation>
     </message>
     <message>
         <location line="-809"/>
         <source>Octave Release Notes</source>
-        <translation type="unfinished"></translation>
+        <translation>Versionshinweise zu Octave</translation>
     </message>
     <message>
         <location line="+157"/>
         <source>Set working directory</source>
         <translation>Arbeitsverzeichnis setzen</translation>
     </message>
     <message>
         <location line="+736"/>
@@ -1066,27 +1074,27 @@ zu: </translation>
     <message>
         <location line="+48"/>
         <source>Show File Browser</source>
         <translation>Dateibrowser anzeigen</translation>
     </message>
     <message>
         <location line="+12"/>
         <source>Show News Window</source>
-        <translation type="unfinished"></translation>
+        <translation>Fenster mit Neuigkeiten anzeigen</translation>
     </message>
     <message>
         <location line="+11"/>
         <source>File Browser</source>
         <translation>Dateibrowser</translation>
     </message>
     <message>
         <location line="+12"/>
         <source>News</source>
-        <translation type="unfinished"></translation>
+        <translation>Neuigkeiten</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Reset Default Window Layout</source>
         <translation>Fensterlayout auf Grundeinstellung zurücksetzen</translation>
     </message>
     <message>
         <location line="+86"/>
@@ -1116,27 +1124,27 @@ zu: </translation>
     <message>
         <location line="+3"/>
         <source>Online</source>
         <translation>Im Internet</translation>
     </message>
     <message>
         <location line="+12"/>
         <source>&amp;News</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Neuigkeiten</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Release Notes</source>
-        <translation type="unfinished"></translation>
+        <translation>Versionshinweise</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Community News</source>
-        <translation type="unfinished"></translation>
+        <translation>Neuigkeiten der Gemeinschaft</translation>
     </message>
     <message>
         <location line="+30"/>
         <source>Enter directory name</source>
         <translation>Neues Verzeichnis eingeben</translation>
     </message>
     <message>
         <location line="+8"/>
@@ -1225,17 +1233,17 @@ zu: </translation>
         <translation>Fehler melden</translation>
     </message>
 </context>
 <context>
     <name>news_dock_widget</name>
     <message>
         <location filename="../src/news-dock-widget.cc" line="+44"/>
         <source>Community News</source>
-        <translation type="unfinished"></translation>
+        <translation>Neuigkeiten der Gemeinschaft</translation>
     </message>
 </context>
 <context>
     <name>octave_dock_widget</name>
     <message>
         <location filename="../src/octave-dock-widget.cc" line="+56"/>
         <location line="+159"/>
         <source>Undock widget</source>
@@ -1252,32 +1260,32 @@ zu: </translation>
         <translation>Fenster andocken</translation>
     </message>
 </context>
 <context>
     <name>octave_qscintilla</name>
     <message>
         <location filename="../src/m-editor/octave-qscintilla.cc" line="+127"/>
         <source>Help on</source>
-        <translation type="unfinished"></translation>
+        <translation>Hilfe zu</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Documentation on</source>
-        <translation type="unfinished"></translation>
+        <translation>Dokumentation zu</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Edit</source>
-        <translation type="unfinished"></translation>
+        <translation>Editieren</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>&amp;Run Selection</source>
-        <translation type="unfinished"></translation>
+        <translation>Markierter Quellcode au&amp;sführen</translation>
     </message>
 </context>
 <context>
     <name>octave_qt_link</name>
     <message>
         <location filename="../src/octave-qt-link.cc" line="+270"/>
         <source>The file %1 does not exist in the load path.  To debug the function you are editing, you must either change to the directory %2 or add that directory to the load path.</source>
         <translation>Die Datei %1 exisitiert nicht im Suchpfad. Um die editierte Funktion zu debuggen, muss entweder in das Verzeichnis %2 gewechselt werden oder dieses Verzeichnis dem Suchpfad hinzugefügt werden.</translation>
@@ -1408,17 +1416,17 @@ zu: </translation>
     <message>
         <location line="+61"/>
         <source>Restore editor tabs from previous session on startup</source>
         <translation>Editor Dateien der letzten Sitzung wiederherstellen</translation>
     </message>
     <message>
         <location line="+13"/>
         <source>Create non existing files without prompting</source>
-        <translation type="unfinished"></translation>
+        <translation>Nicht existente Dateien ohne Nachfrage erzeugen</translation>
     </message>
     <message>
         <location line="+47"/>
         <source>Use custom file editor</source>
         <translation>Externen Editor verwenden</translation>
     </message>
     <message>
         <location line="+10"/>
@@ -1684,38 +1692,49 @@ zu: </translation>
     <message>
         <location line="+14"/>
         <source>Welcome to Octave!</source>
         <translation>Willkommen zu Octave!</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>You seem to be using the Octave graphical interface for the first  time on this computer.  Click &apos;Finish&apos; to write a configuration file  and launch Octave GUI.</source>
-        <translation type="unfinished"></translation>
+        <translation>Die grafische Nutzerschnittstelle von Octave wird offenbar das erste Mal gestartet. Ein Klick auf &apos;Beenden&apos; erstellt eine Standard-Konfigurationsdatei und startet Octave GUI.</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>The configuration file is stored in %1. If that file exists, you will not see this dialog when Octave starts again.</source>
-        <translation type="unfinished"></translation>
+        <translation>Die Konfigurationsdatei wird in %1 gespeichert. Wenn diese Datei existiert, erscheint dieser Dialog beim nächsten Start von Octave nicht mehr.</translation>
     </message>
     <message>
         <location line="+23"/>
         <source>&lt;html&gt;&lt;head&gt;
 &lt;style&gt;
 a:link { text-decoration: underline; color: #0000ff; }
 &lt;/style&gt;
 &lt;head/&gt;&lt;body&gt;
 &lt;p&gt;For more information about Octave:&lt;/p&gt;
 &lt;ul&gt;
 &lt;li&gt;Visit &lt;a href=&quot;http://octave.org&quot;&gt;http://octave.org&lt;/a&gt;&lt;/li&gt;
 &lt;li&gt;Get the documentation online as &lt;a href=&quot;http://www.gnu.org/software/octave/doc/interpreter/index.html&quot;&gt;html&lt;/a&gt;- or &lt;a href=&quot;http://www.gnu.org/software/octave/octave.pdf&quot;&gt;pdf&lt;/span&gt;&lt;/a&gt;-document&lt;/li&gt;
 &lt;li&gt;Open the documentation browser of Octave GUI with the help menu&lt;/li&gt;
 &lt;/ul&gt;
 &lt;/body&gt;&lt;/html&gt;</source>
-        <translation type="unfinished"></translation>
+        <translation>&lt;html&gt;&lt;head&gt;
+&lt;style&gt;
+a:link { text-decoration: underline; color: #0000ff; }
+&lt;/style&gt;
+&lt;head/&gt;&lt;body&gt;
+&lt;p&gt;Für mehr Informationen über Octave:&lt;/p&gt;
+&lt;ul&gt;
+&lt;li&gt;Besuchen Sie &lt;a href=&quot;http://octave.org&quot;&gt;http://octave.org&lt;/a&gt;&lt;/li&gt;
+&lt;li&gt;Holen Sie sich die Dokumentation online als &lt;a href=&quot;http://www.gnu.org/software/octave/doc/interpreter/index.html&quot;&gt;html&lt;/a&gt; oder &lt;a href=&quot;http://www.gnu.org/software/octave/octave.pdf&quot;&gt;pdf&lt;/span&gt;&lt;/a&gt;-Dokument&lt;/li&gt;
+&lt;li&gt;Öffnen Sie den Dokumentations-Browser von Octave GUI im Hilfe-Menü&lt;/li&gt;
+&lt;/ul&gt;
+&lt;/body&gt;&lt;/html&gt;</translation>
     </message>
     <message>
         <location line="+20"/>
         <source>Finish</source>
         <translation>Beenden</translation>
     </message>
 </context>
 <context>
